### PR TITLE
added base files and generic class with size and dictionary

### PR DESCRIPTION
**Pull request by DanerSound**

### CODE REVIEW

File: keyGenerartor/src/base/KeyGenerartor.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package base;
+
+import java.util.Random;
+
+public class KeyGenerartor {
```
Comment (Wabri) [inline keyGenerartor/src/base/KeyGenerartor.java]: The name of the class is mistyped, should be:
```suggestion
public class KeyGenerator {
```

File: keyGenerartor/src/base/MainTest.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
+package base;
+
+public class MainTest {
+
+	public static void main(String[] args) {
+		
+		String[] fileNames = {"chiave2.txt", "chiave4.txt","chiave8.txt","chiave16.txt"};
+		String[] key;
```
Comment (Wabri) [inline keyGenerartor/src/base/MainTest.java]: The key can also be a pure String type and it's more simply to use with the operator like +:
`key += letter;`
instead:
`key[index]=letter.toString();`

Comment (DanerSound) [reply keyGenerartor/src/base/MainTest.java]: the method "KeyGenerator" returns an array so, i can't  do that, also when i use a simple string i have problems with the first value

Comment (Wabri) [reply keyGenerartor/src/base/MainTest.java]: What problem?

Comment (Wabri) [reply keyGenerartor/src/base/MainTest.java]: see PR #9

File: keyGenerartor/src/base/KeyGenerator.java [hunk 3]
```diff
@@ -4,12 +4,12 @@
...
 
-	public static String[] Generator(Integer keyLenght, String currentDictionary) {
-		String[] key = new String[keyLenght];
+	public static String Generator(Integer keyLenght, String currentDictionary) {
+		String key = "";
 
 		for (int index = 0; index < keyLenght; index++) {
 			Character letter = currentDictionary.charAt((new Random()).nextInt(currentDictionary.length()));
-			key[index] = letter.toString();
+			key += letter.toString();
```
Comment (Wabri) [inline keyGenerartor/src/base/KeyGenerator.java]: toString is not necessary with the last changes I think:
```suggestion
			key += letter;
```

File: keyGenerartor/src/base/WriteOnFile.java [hunk 4]
```diff
@@ -5,16 +5,17 @@
...
+	public static void writeKey(String keys, String fileName) {
 		BufferedWriter bw = null;
 		FileWriter fw = null;
 		try {
 			fw = new FileWriter(fileName);
 			bw = new BufferedWriter(fw);
-			for (int i = 0; i < keys.length; i++) {
-				  bw.write(keys[i]);	
+			for(int i=0; i<keys.length(); i++) {
+			bw.write(keys.charAt(i));
```
Comment (Wabri) [inline keyGenerartor/src/base/WriteOnFile.java]: Indentation

**Pull request merged**